From 497d03af2651289c70acacfe3188424d2621d01b Mon Sep 17 00:00:00 2001
From: Nic Costa <nic.costa@arm.com>
Date: Mon, 16 Oct 2017 13:23:35 -0500
Subject: [PATCH 2/2] downrev FlashIAP driver in mbed-bootloader/mbed-os

fixes the slow write problem during fota

to enable the correct FlashIAP driver in mbed-os, we need
to set "FLASH_CMSIS_ALGO" on our target.

PATCH 2 which modifies the FlashIAP driver is applied to
mbed-bootloader/mbed-os

Signed-off-by: Nic Costa <nic.costa@arm.com>
---
 mbed_app.json | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mbed_app.json b/mbed_app.json
index d0d582e..2924f13 100755
--- a/mbed_app.json
+++ b/mbed_app.json
@@ -44,6 +44,7 @@
             "firmware_metadata_header_size": "0x400"
         },
         "UBLOX_EVK_ODIN_W2": {
+            "target.extra_labels_add": ["STM32F4", "STM32F439", "STM32F439ZI","STM32F439xx", "STM32F439xI", "FLASH_CMSIS_ALGO"],
             "firmware_metadata_header_address": "0x08020000",
             "update-client.application-details": "0x08020000",
             "firmware_metadata_header_size": "0x400"
-- 
2.14.2

